pkg/skaffold/initializer/build/resolve.go

```diff
@@ -53,12 +53,47 @@ func (d *defaultBuildInitializer) resolveBuilderImages() error {
 	}
 
 	if d.force {
-		return errors.BuilderImageAmbiguitiesErr{}
+		return d.resolveBuilderImagesForcefully()
 	}
 
 	return d.resolveBuilderImagesInteractively()
 }
 
+func (d *defaultBuildInitializer) resolveBuilderImagesForcefully() error {
+	// In the case of 1 image and multiple builders, respects the ordering Docker > Jib > Bazel > Buildpacks
+	if len(d.unresolvedImages) == 1 {
+		image := d.unresolvedImages[0]
+		choice := d.builders[0]
+		for _, builder := range d.builders {
+			if builderRank(builder) < builderRank(choice) {
+				choice = builder
+			}
+		}
+
+		d.builderImagePairs = append(d.builderImagePairs, BuilderImagePair{Builder: choice, ImageName: image})
+		d.unresolvedImages = util.RemoveFromSlice(d.unresolvedImages, image)
```

Comment:
you could set `d.unresolvedImages` to empty array instead of calling `RemoveFromSlice` here.